pkg/workflows/wasm/test/cmd/main.go

```diff
@@ -24,6 +24,8 @@ func BuildWorkflow(config []byte) *workflows.WorkflowSpecFactory {
 
 func main() {
 	runner := sdk.NewRunner()
+	runner.SDK.LogInfo("building workflow...")
 	workflow := BuildWorkflow(runner.Config())
+	runner.SDK.LogInfo("running workflow...")
 	runner.Run(workflow)
```

Comment:
Could we also add some debug logging to the runner itself as part of this? I think it would help speed up debugging in the future and would generally be very useful.